<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>Plot Tool Bar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="linear_region_action"/>
   <addaction name="remove_region_action"/>
   <addaction name="separator"/>
   <addaction name="change_color_action"/>
   <addaction name="separator"/>
   <addaction name="reset_view_action"/>
   <addaction name="export_plot_action"/>
  </widget>
  <action name="linear_region_action">
   <property name="icon">
    <iconset resource="../../data/resources/resources.qrc">
     <normaloff>:/icons/region_add.svg</normaloff>:/icons/region_add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Region</string>
   </property>
   <property name="toolTip">
    <string>Add a linear selection region</string>
   </property>
  </action>
  <action name="rectangular_region_action">
   <property name="text">
    <string>Rectangular Region</string>
   </property>
   <property name="toolTip">
    <string>Add a rectangular selection region</string>
   </property>
  </action>
  <action name="plot_options_action">
   <property name="text">
    <string>Plot Options</string>
   </property>
   <property name="toolTip">
    <string>Options for displaying plotted data</string>
   </property>
  </action>
  <action name="export_plot_action">
   <property name="icon">
    <iconset resource="../../data/resources/resources.qrc">
     <normaloff>:/icons/export.svg</normaloff>:/icons/export.svg</iconset>
   </property>
   <property name="text">
    <string>Export Plot</string>
   </property>
   <property name="toolTip">
    <string>Export the currently display plot</string>
   </property>
  </action>
  <action name="remove_region_action">
   <property name="icon">
    <iconset resource="../../data/resources/resources.qrc">
     <normaloff>:/icons/region_remove.svg</normaloff>:/icons/region_remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Region</string>
   </property>
   <property name="toolTip">
    <string>Remove selected region</string>
   </property>
  </action>
  <action name="change_color_action">
   <property name="icon">
    <iconset resource="../../data/resources/resources.qrc">
     <normaloff>:/icons/color.svg</normaloff>:/icons/color.svg</iconset>
   </property>
   <property name="text">
    <string>Change Color</string>
   </property>
   <property name="toolTip">
    <string>Change the current plot item color</string>
   </property>
  </action>
  <action name="reset_view_action">
   <property name="icon">
    <iconset resource="../../data/resources/resources.qrc">
     <normaloff>:/icons/house.svg</normaloff>:/icons/house.svg</iconset>
   </property>
   <property name="text">
    <string>Reset View</string>
   </property>
   <property name="toolTip">
    <string>Reset the axes to fit the data</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../data/resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
